CIP-XXX/CIP-XXX.md

```diff
@@ -27,10 +27,53 @@ As part of this, we also describe the bit ordering within a `BuiltinByteString`,
 and provide some laws these operations should obey.
 
 ## Motivation: why is this CIP necessary?
+Currently, Plutus Core has a `BuiltinByteString` type, which is a packed array of bytes, similar to Haskell's ByteString.
+We want to implement bit wise operations on `BuiltinByteString` type. In otherwords, we want to achieve bit level granularity on the operations on `BuiltinByteString`.
```

Comment:
This is too abrupt as a lead-in. The aim of this section is to _motivate_ what we want to do: see [the opening paragraph here](https://github.com/mlabs-haskell/CIPs/tree/koz/to-from-bytestring/CIP-XXXX#motivation-why-is-this-cip-necessary) for a better idea of what kind of wording and reasoning would make sense.